common/Cargo.toml

```diff
@@ -11,6 +11,7 @@ script = { path = '../script'}
 crypto = { path = '../crypto'}
 serialization = { path = "../serialization" }
 logging = { path = "../logging/" }
+utils = { path = "../utils/" }
```

Comment:
I have a feeling this isn't used anywhere?

Reply:
Well spotted. I used `ensure` somewhere, removed it later but forgot about the dep.